cmake/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(colvars CXX)
+get_filename_component(COLVARS_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/.. ABSOLUTE)
+
+#set(CMAKE_VERBOSE_MAKEFILE on)
+
+option(ENABLE_LEPTON "Build and link the Lepton library" OFF)
```

Comment:
Could this default to ON if Lepton sources are found?

Reply:
I preferred not to have this as the standard arrangement, since one of the key selling points of CMake are out-of-source builds.  I was able, however, to find out how to handle the commandline-provided path (turned out to be trivial).